Packs/GitHub/Integrations/GitHub/README.md

```diff
@@ -2848,3 +2848,26 @@ Deletes a comment.
 #### Context Output
 
 There is no context output for this command.
+
+### GitHub-add-assignee
+***
+Adds up to 10 assignees to an issue. Users already assigned to an issue are not replaced.
+
+
+#### Base Command
+
+`GitHub-add-assignee`
+#### Input
+
+| **Argument Name** | **Description**                                          | **Required** |
+|-------------------|----------------------------------------------------------|--------------|
```

Comment:
why is this so big? you can reduce to 3 `-`

Reply:
changed.
It wasn't like that when I started, not sure how it happened.